redirects/pages.yml

```diff
@@ -756,3 +756,23 @@
     - /docs/kmm-understand-project-structure.html
     - /docs/mpp-discover-project.html
   to: /docs/multiplatform-discover-project.html
+
+- from:
```

Comment:
@zoobestik shouldn't these be implemented via "accept-web-file-names" in kr.tree?

Reply:
IMO, there is not a big difference between both ways.
If it's difficult to read `kr.tree`, it's okay use `redirects/*.yml`, up to you.